packages/core/saved-objects/core-saved-objects-import-export-server-internal/src/import/import_saved_objects.ts

```diff
@@ -49,6 +49,11 @@ export interface ImportSavedObjectsOptions {
   namespace?: string;
   /** If true, will create new copies of import objects, each with a random `id` and undefined `originId`. */
   createNewCopies: boolean;
+  /**
+   * If true, Kibana will apply various adjustments to the data that's being imported to maintain compatibility between
+   * different Kibana versions (e.g. generate legacy URL aliases for all imported objects that have to change IDs).
+   */
+  compatibilityMode: boolean;
```

Comment:
__note:__ I went back and forth trying to find the appropriate name for this parameter without making a notion of the legacy URL alias a part of the public contract, and ended up with something that's close to what we really do (maintain not-officially-supported "weak-links" behavior) and vague enough to not expose implementation detail. I'd be happy to accept any other better name suggestion though.